docs/configuration/settings.rst

```diff
@@ -161,6 +161,114 @@ Campaign settings
 Email settings
 **************
 
+Email transport settings
```

Comment:
@escopecz I figured that email transport settings might be a more accurate heading, WDYT?